new-lamassu-admin/src/components/LogsDownloaderPopper.js

```diff
@@ -132,36 +131,26 @@ const LogsDownloaderPopover = ({ id, open, anchorEl, onClose, logsResponse, ...p
     [classes.dateRangePickerHidden]: radioButtons === 0
   }
 
-  const formatDateFile = date => {
-    return moment(date).format('YYYY-MM-DD_HH-mm')
-  }
-
   const handleRadioButtons = (event) => {
-    setRadioButtons(toInteger(event.target.value))
+    const radio = toInteger(event.target.value)
+    setRadioButtons(radio)
+    if (radio === 0) setRange({ from: null, to: null })
```

Comment:
why not just use boolean?

Reply:
The RadioGroup component is prepared for more than 2 options. Using the `value` identifier even when only two options are used is more consistent.

Reply:
better to have name values, then.

Reply:
Fixed in https://github.com/lamassu/lamassu-server/pull/336/commits/58c08e39d6c46210864dc97c5ae25ec7fae489b7